trino/trino.go

```diff
@@ -1043,6 +1115,30 @@ func (qr *driverRows) initColumns(qresp *queryResponse) error {
 	return nil
 }
 
+func (qr *driverRows) updateCallback(id string, stats stmtStats) {
```

Comment:
would `scheduleQueryProgressCallbackIfNeeded` be a better name here?

Reply:
Hmm... seems to be quite long for function name, so how about `scheduleProgressUpdate` or `updateProgress`?

Reply:
`scheduleProgressUpdate` is fine